### PR TITLE
feat(create-cluster): summary step

### PR DESCRIPTION
**Pull request by RemiBonnet**
# What does this PR do?

[> Link to the JIRA ticket
](https://qovery.atlassian.net/jira/software/projects/FRT/boards/23?selectedIssue=FRT-580)

- Added last step summary for Cluster
- Fixed step indicator
- Fixed navigation

Capture d’écran 2023-02-07 à 17 55 43

---

## PR Checklist

- [ ] This PR introduces breaking change(s) and has been labeled as such
- [ ] This PR introduces new store changes
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`
- [x] I made sure the code is type safe (no any)

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.
Click on the link below to follow its deployment and use it.
:point_right: [[PR] staging - feat(create-cluster): summary step - 2023-02-06T16:40:11Z](https://new.console.qovery.com/organization/141c07c8-0dd9-4623-983b-3fdd61867255/project/977bd069-20f7-4b24-bd95-28b44174cf43/environment/6788992b-37e6-4d84-802e-f76b5656fc54/services/general)

 Another comment will be posted when deployments are terminated

**Comment by evoxmusic** [comment]
Your preview environment has been successfully deployed !
Click on the link below to open your service:
:point_right: [console](https://z6788992b-z77634983-gtw.za8ad0659.bool.sh)
:point_right: [storybook](https://z6788992b-zdea74763-gtw.za8ad0659.bool.sh)

**Review by bdebon — APPROVED** [review]
Good!

**Comment by bdebon** [comment]
So cool to finally see the whole flow! 🤩
Good job plugging all the steps together.
I have not checked the code yet, but did this first QA and had the issues below.

This aws Aws is a little redundant
CleanShot 2023-02-08 at 11 51 15@2x

I set my cluster as KES, went until summary, then came back to resources, set my cluster to the other, then I had to specify the feature, but in the final summary, I still have the ssh remote key and not the feature displayed
CleanShot 2023-02-08 at 11 53 12@2x

Same when I do this opposite, I make an EKS, fill up the feature, go to summary, and then if I come back to resources and change to KES, I fill the ssh, and then I have both feature and ssh at the end.
CleanShot 2023-02-08 at 11 56 34@2x

I tried to create a cluster on EKS AWS with VPC Subnet feature checked and I got this error. Maybe the form should not allow to go to next step if we have not selected a vpc subnet address value
CleanShot 2023-02-08 at 11 57 31@2x

Maybe the default nodes value should be changed because we have this error if we try to create a cluster with default value:
CleanShot 2023-02-08 at 11 59 22@2x

When I checked the credentials value for a newly created cluster (not installed), I got this error: 
CleanShot 2023-02-08 at 12 00 39@2x
Maybe it's normal as long as it's not installed?

File: libs/pages/clusters/src/lib/feature/page-clusters-create-feature/page-clusters-create-feature.tsx [hunk 1]
```diff
@@ -36,13 +37,31 @@ export const useClusterContainerCreateContext = () => {
...
-export const steps: { title: string }[] = [
-  { title: 'Create new cluster' },
-  { title: 'Set resources' },
-  { title: 'Set SSH Key' },
-  { title: 'Set features' },
-  { title: 'Ready to install' },
-]
+export const steps = (cloudProvider?: CloudProviderEnum, clusterType?: string) => {
+  if (cloudProvider === CloudProviderEnum.SCW) {
+    return [
```
Comment (bdebon) [inline libs/pages/clusters/src/lib/feature/page-clusters-create-feature/page-clusters-create-feature.tsx:42]: Smart way to take care of that! Gg!

**Review by bdebon — COMMENTED** [review]
Code is perfect! I just added some QA feedback on another comment

**Comment by bdebon** [comment]
I don't know exactly what I did, but in this case, AWS, with EKS, and in this case, I don't see at all the summary of features.
Alors, be careful, it's written : `Ready to install your database`

CleanShot 2023-02-08 at 12 10 01@2x

**Comment by bdebon** [comment]
When I go back to the features page, the checkboxes are not selected anymore:
CleanShot 2023-02-08 at 12 13 53@2x

**Pull request merged**